navigation/Home/HomeTabNavigator.tsx

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react-native/no-unused-styles */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable react/prop-types */
+/* eslint-disable react-native/no-inline-styles */
+/* eslint-disable react-native/no-color-literals */
+import * as React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { View, Image, TouchableOpacity, StyleSheet } from 'react-native';
+import {OnboardingScreen} from '../../screens/Onboarding/OnboardingScreen';
+import {MyProfileScreen} from '../../screens/Profile/MyProfileScreen/MyProfileScreen';
+import {AuthenticationScreen} from '../../screens/Registration/AuthenticationScreen/AuthenticationScreen';
+//lib for react-native-svg
+import { Svg, Path } from 'react-native-svg';
+import { SvgProps } from 'react-native-svg';
+
+// Import your tab icons
+import { white, peach, tabBarFill } from '../../assets/colors/colors';
+
+import HomeIconFocused from '../../assets/icon-home-focused.svg';
+//import HomeIconUnfocused from '../../assets/icon-home-unfocused.svg';
+import ExploreIconFocused from '../../assets/icon-explore-focused.svg';
+import ExploreIconUnfocused from '../../assets/icon-explore-unfocused.svg';
+import AddIcon from '../../assets/add-icon.svg';
```

Comment:
You probably missed this when we talked about this in-person, but importing a `.png` or a `.svg` for an icon is not efficient. You should install and use `@expo/vector-icons` package which provides you with many icons out of the box. Also, all of the icons are stylable. You can find all icons listed [here](https://icons.expo.fyi/Index). E.g. you would probably want to use https://icons.expo.fyi/Index/Ionicons/location-outline from Ionicons here.

Reply:
Thanks for the feedback it's done